modules/use-process-baselines.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operating/evaluate-security-risks.adoc
+:_module-type: CONCEPT
+[id="use-process-baselines_{context}"]
+= Using process baselines
+
+[role="_abstract"]
+In {product-title}, you can minimize your risks by using process baselining.
+It is a proactive approach to keep your infrastructure secure.
```

Comment:
Can you combine the first three sentences for clarity, perhaps something like, "You can minimize risk by using process baselining for infrastructure security. With this approach, RHACS first discovers...